src/debug/_debug-mixins.scss

```diff
@@ -0,0 +1,122 @@
+// DEBUG: MIXINS
```

Comment:
very very cool
maybe interesting task to add a pipeline rule to fail if these fail